src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+import { People } from '../components/People';
+import { Loader } from '../components/Loader';
+import { Person } from '../types';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState<string | null>(null);
+  const [people, setPeople] = useState<Person[] | null>(null);
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      try {
+        setIsLoading(true);
+        setError(null);
+
+        const response = await fetch(
+          'https://mate-academy.github.io/react_people-table/api/people.json',
```

Comment:
Ensure that the URL for fetching the people data is correct and accessible. If the URL changes or is incorrect, it will result in a fetch error.